### DISCUSSION TITLE
71361762-3625-4E90-A8C6-2BA14BE3B665

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#49] [General]
# 71361762-3625-4E90-A8C6-2BA14BE3B665

GJ 4332, also known as L 168-9, is a red dwarf star about 82.09 light-years away in the constellation Tucana. A superterran planet about 4.6 times more massive than Earth was discovered in a 1.4-day orbit around its star. Both objects will receive formal names at the end of the third NameExoWorlds campaign, and will be one of the first exoplanet systems that the JWST will observe and study

https://celestia.mobi/addon?item=71361762-3625-4E90-A8C6-2BA14BE3B665&lang=zh_CN

**Comment by 13940951572** [comment]
希望能出一颗GJ504b粉色星球
